sample.js

```diff
@@ -0,0 +1,11 @@
+const asyncfunction = async() => {
+    const Post = ('I have joy')
```

Comment:
Hello @moyinmi, what does this code do? You can rewrite this as 

`const postText = 'I have joy'`